test/test-app.js

```diff
@@ -18,11 +18,9 @@ function exec(command, args) {
   return execResult.stdout.toString();
 }
 
-function setupGit(dir, done) {
+function addFile(dir, done) {
   var fileName = path.join(dir, 'index.html');
   fs.writeFile(fileName, '', done);
-  exec('git', 'config user.email "you@example.com"');
-  exec('git', 'config user.name "Your Name"');
```

Comment:
maybe we can have this configuration locally and remove global configuration from travis yaml instead?